docs/user/data_management.rst

```diff
@@ -160,6 +160,36 @@ If you are using the GeoMesa ``SftBuilder``, you may call the overloaded attribu
         .geometry("geom", default = true)
         .build("mySft")
 
+.. _configuring_z_shards:
+
+Configuring Z-Index Shards
+--------------------------
+
+GeoMesa Accumulo permits one to specify the number of shards (or splits) into which the Z2 (or Z3) indices are
+divided. This parameter may be changed individually for each ``SimpleFeatureType``. In previous versions of GeoMesa,
+the numberof shards was "hard-coded" to 4, which remains the default. Valid values for the number of shards range
+from 1 to 127 inclusive.
+
+When using very large data sets or large clusters, it may be of benefit to use a larger number of shards. However, in
+most test cases--especially with single-node or small clusters--the default of 4 should be preferred.
+
+The number of shards is set when calling ``createSchema``. It may be specified through the simple feature type
+user data using the hint ``geomesa.z.splits``:
+
+.. code-block:: java
+
+    // set the hint directly
+    SimpleFeatureType sft = ...
+    sft.getUserData().put("geomesa.z.splits", "16");
+
+.. code-block:: java
+
+    // or through using RichSimpleFeatureType
+    import org.locationtech.geomesa.utils.geomesa.RichSimpleFeatureType.RichSimpleFeatureType;
+
+    SimpleFeatureType sft = ...
+    sft.setZShards(16);
```

Comment:
Did you try this out? i doubt java would work with the implicit wrappers here. I think we can remove this code block, the implicit conversion is more for internal use.